bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -57,13 +58,16 @@
 
 	@Parameter(defaultValue = "${project.build.outputDirectory}", readonly = true)
 	private File classesDir;
-	
+
 	@Parameter(defaultValue = "${project.build.outputDirectory}/META-INF/MANIFEST.MF", readonly = true)
 	private File manifestPath;
-	
+
 	@Parameter(defaultValue = "${project}", required = true, readonly = true)
 	private MavenProject project;
-	
+
+	@Component
+	private BuildContext buildContext;
```

Comment:
Another question... is there any time that you can think of that we wouldn't want to rerun the manifest generation based on an Eclipse workspace build?  If so we can use this API.  I figured that we want to rebuild the Manifest for every single Eclipse build just to keep things simple.